src/util/report-error.js

```diff
@@ -55,7 +55,7 @@ export default async (sentry, error, apiUrl, configFiles) => {
         for (const flag of flags) {
           if (args[flag]) args[flag] = 'REDACTED';
         }
-        if (args._[0].startsWith('secret') && args._[1] === 'add' && args._[3]) {
+        if (args._[0] && args._[0].startsWith('secret') && args._[1] === 'add' && args._[3]) {
```

Comment:
_could_ be shortened to

```suggestion
        if (args._[3] && args._[0].startsWith('secret') && args._[1] === 'add') {
```

or

```suggestion
        if (args._.length >= 4 && args._[0].startsWith('secret') && args._[1] === 'add') {
```